simpleMail.py

```diff
@@ -67,11 +67,15 @@ def send_email(frm, to, subject, message, attachments=None, maxsize=100000, html
         mimetype = mimetypes.guess_type(attachment)[0]
         _maintype, _subtype = mimetype.split('/')
         att = MIMEBase(_maintype, _subtype)
-        att.set_payload(open(attachment, 'rb').read())
-        encoders.encode_base64(att)
-        file_name=os.path.basename(attachment)
-        att.add_header('Content-Disposition', 'attachment', filename=file_name)
-        msg.attach(att)
+        try:
+            att.set_payload(open(attachment, 'rb').read())
+        except OSError as err:
+            print('OS Error: {}'.format(err))
```

Comment:
I just noticed that I could put `msg.attach(att)` in a separate `finally:` block.